### DISCUSSION TITLE
Composer Packagist Integration

### DISCUSSION THREAD

**Discussion by zngly-vlad** [wpengine/atlas-content-modeler#525] [Ideas]
As a developer, I would like to install/update my plugins via composer. This would allow me to version control and manage plugins without using the admin ui. This also makes managing code through github much easier to maintain since you dont have to upload the entire plugin.

**Comment by mindctrl** [comment]
@ditto-vlad have you tried using WordPress Packagist? https://wpackagist.org/search?q=atlas-content-modeler&type=any&search=

**Comment by zngly-vlad** [comment]
Didn't know there was a wordpress packagist, maybe this could also be included in the readme, as part of how to install ACM, or where to find it.

**Reply by mindctrl** [reply]
Good idea! I'll make a note to add it to our readme and docs site.

**Reply by zngly-vlad** [reply]
Awesome!
